python/torch_mlir_e2e_test/linalg_on_tensors_backends/refbackend.py

```diff
@@ -65,6 +65,42 @@ def consume_return_f32(a):
         def consume_return_f64(a):
             self.result = a
 
+        @ctypes.CFUNCTYPE(None, ctypes.POINTER(UnrankedMemRefDescriptor),
+                          ctypes.POINTER(UnrankedMemRefDescriptor))
+        def consume_return_mrf32_mri64(arg0, arg1):
+            self.result = unranked_memref_to_numpy(
+                arg0, np.float32), unranked_memref_to_numpy(
+                    arg1,
+                    np.int64)
+
+        self.ee.register_runtime("refbackend_consume_func_return_mri1",
+                                 consume_return_mri1)
+
+        self.ee.register_runtime("refbackend_consume_func_return_mri32",
+                                 consume_return_mri32)
+
+        self.ee.register_runtime("refbackend_consume_func_return_mri64",
+                                 consume_return_mri64)
+
+        self.ee.register_runtime("refbackend_consume_func_return_mrf32",
+                                 consume_return_mrf32)
+
+        self.ee.register_runtime("refbackend_consume_func_return_mrf64",
+                                 consume_return_mrf64)
+
+        self.ee.register_runtime("refbackend_consume_func_return_i64",
+                                 consume_return_i64)
+
+        self.ee.register_runtime("refbackend_consume_func_return_f32",
+                                 consume_return_f32)
+
+        self.ee.register_runtime("refbackend_consume_func_return_f64",
+                                 consume_return_f64)
```

Comment:
I think the CI error is related to these. These are already present below. You should only need to add the `consume_return_mrf32_mri64`.